mypy/nodes.py

```diff
@@ -2053,7 +2053,7 @@ def is_generic(self) -> bool:
         """Is the type generic (i.e. does it have type variables)?"""
         return len(self.type_vars) > 0
 
-    def get(self, name: str) -> 'SymbolTableNode':
+    def get(self, name: str) -> Optional['SymbolTableNode']:
```

Comment:
This seems to be OK, grep shows dozens of this pattern elsewhere, so I didn't change this.
Also IIRC the union problem was specific _only_ to import cycles, normal forward references were always OK. (Btw this is just merge from master, not something related to protocols.)